doc/ArchInstall/arch-chroot_install.sh

```diff
@@ -75,16 +74,12 @@ pacman -Syu
 pacman -S --noconfirm --needed \
 	wpa_supplicant \
 	openssh \
-	vim \
-	nano \
 	wget \
-	screen \
-	htop \
```

Comment:
I like `htop` :(

Reply:
The idea is all these package install stuff gets moved to this module rather than being a part of the general arch install. That way you can install the literal bare minimum for the OS install, and this module deals with getting everything else and setting up the system

Reply:
https://github.com/NUbots/NUbots/pull/299/files#diff-b7f20886fac39e2b0fda7e357a4330dc797c0920d10bd9a3b8614cfcdee9ee3bR22

See, all safe and sound